File: exam-system-java/entity/src/main/java/com/yyjy/exam/entity/question/io/res/QuestionPageRes.java
package com.yyjy.exam.entity.question.io.res;

import com.yyjy.exam.entity.question.dto.QuestionsPageView;
import lombok.Builder;

import java.util.List;

@Builder
public record QuestionPageRes(
    List<QuestionsPageView> records,
    long total,
    int current,
    int size,
    long pages
) {}

File: exam-system-server/src/main/kotlin/com/yyjy/models/res/QuestionPageRes.kt
package com.yyjy.models.res

import com.yyjy.models.entity.dto.QuestionsPageView

/**
 * @author Nyxcirea
 * @date 2026/4/9
 * @description: TODO
 */
data class QuestionPageRes(
    val records: List<QuestionsPageView>,
    val total: Long,
    val current: Int,
    val size: Int,
    val pages: Long
)